scripts/util_functions.sh

```diff
@@ -469,7 +472,7 @@ install_magisk() {
   cd $MAGISKBIN
 
   if [ ! -c $BOOTIMAGE ]; then
-    eval $BOOTSIGNER -verify < $BOOTIMAGE && BOOTSIGNED=true
+    [ -f /system/bin/dalvikvm ] && eval $BOOTSIGNER -verify < $BOOTIMAGE && BOOTSIGNED=true
```

Comment:
AVB 1.0 is only required for very old devices, and they are unlikely to have LOS Recovery. I recommend skipping mount apex and exec dalvikvm for new devices

Reply:
That would be another option yeah

Reply:
Many old devices have Lineage Recovery however.

Reply:
oh, I thought it came after the A/B device. So, is there any other way to skip for avb 2.0 devices?

Reply:
Yeah there's definitely A-only Lineage Recovery too for some devices. Could we somehow grab from magiskboot when it's AVB 2.0 and skip based on that?

Reply:
I'm thinking I could grab the AVB status using `tee` if you think that could be a good way to go about it.

Reply:
Added skipping apex mounts entirely when /system/bin/dalvikvm isn't present nicely excludes all the latest Android versions and devices, so I think we can consider this resolved?